### PR TITLE
#949 disable stored payment

### PR DESCRIPTION
**Pull request by lojzatran**
Addressing: https://github.com/commercetools/commercetools-adyen-integration/issues/949

Hint for review: please check the JIRA ticket and understand the problem first.

### CODE REVIEW

File: extension/src/service/web-component-service.js [hunk 1]
```diff
@@ -192,6 +202,10 @@ async function fetchAsync(
   requestObj,
   headers
 ) {
+  const moduleConfig = config.getModuleConfig()
+  const removeSensitiveData =
+    requestObj.removeSensitiveData ?? moduleConfig.removeSensitiveData
+  delete requestObj.removeSensitiveData
```
Comment (leungkinghin) [inline extension/src/service/web-component-service.js:208]: Any reason to remove it from request object?

**Review by leungkinghin — CHANGES_REQUESTED** [review]

File: extension/docs/DisableStoredPayments.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+A shopper may request to delete their saved details for a certain payment method.
+
+### Make an API call to disable a stored payment
+
+To disable a stored payment, create a new payment with `amountPlanned=0` and the `disableStoredPaymentRequest` custom field.
+
+```json
+{
+  "amountPlanned": {
```
[the remark this answers is not in the document]
Comment (lojzatran) [reply extension/docs/DisableStoredPayments.md:22]: We do not need it, but it's a required field. Therefore you cannot omit it: https://docs.commercetools.com/api/projects/payments#paymentdraft

File: extension/src/service/web-component-service.js (hunk 1 above)
Comment (lojzatran) [reply extension/src/service/web-component-service.js:208]: The requestObj is later sent to adyen. I don't want to send them unnecessary data.

File: README.md [hunk 3]
```diff
@@ -25,6 +25,7 @@
 - Authorisation [cancellation](./extension/docs/CancelPayment.md) on a payment that has not yet been captured.
 - [Manual capture](./extension/docs/ManualCapture.md) of a payment.
 - [Restore](./extension/docs/Restore.md), which gives your shoppers an opportunity to offset their carbon emissions from the delivery or lifecycle of their purchase at checkout. 
+- [Store payment](./extension/docs/Restore.md) and [disable stored payment](./extension/docs/DisableStoredPayments.md) gives your shoppers the ability to store payments for later use and removing these stored payments.
```
Comment (lojzatran) [inline README.md:28]:
```suggestion
- [Store payment](./extension/docs/StorePayment.md) and [disable stored payment](./extension/docs/DisableStoredPayments.md) gives your shoppers the ability to store payments for later use and removing these stored payments.
```

[the remark this answers is not in the document]
Comment (lojzatran) [reply README.md]: https://github.com/commercetools/commercetools-adyen-integration/pull/1004/commits/645c6660d655eaceef37e9fcbf6b694f5827df4c

**Review by leungkinghin — APPROVED** [review]

**Pull request merged**